relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -913,6 +1042,15 @@ mod tests {
                     "timestamp": 1597976302.0000000,
                     "trace_id": "ff62a8b040f340bda5d830223def1d81"
                 },
+                {
+                    "description": "<SomeUiRendering>",
+                    "op": "ui.react.render",
```

Comment:
The negative test is the span right above this one. I didn't reuse it to have a negative test, and because changing the op (adding the `ui.` prefix) extracts a measurement less. We should revisit this edge case when we move to onlyspans.